### PR TITLE
Azure Wrapper

### PR DESCRIPTION
**Pull request by tgandrews**
This seems to work for me and resolves issue #3

### CODE REVIEW

**Comment by rnicholus** [comment]
Thanks! A few things before this can proceed:

- [ ] Documentation updates to the README.md file, similar to the S3 & Traditional sections.
- [ ] Can you post here specifics about your Azure setup, or at least your client-side code? I'm a bit more careful with Azure-related updates since I don't use Azure myself.
- [ ] We'll need to release a new version, but I'll take care of that myself last.

**Comment by tgandrews** [comment]
RE: Documentation. Is it ok to add links to the core fine-uploader areas? I'll add my working example as well.

**Comment by rnicholus** [comment]
Please do link to any areas that you feel are useful. But it should at least follow the same format & tone as the existing [S3](https://github.com/FineUploader/react-fine-uploader#s3) and [Traditional](https://github.com/FineUploader/react-fine-uploader#traditional) sections in the readme.

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ These wrap a Fine Uploader instance for use in React Fine Uploader. They provide
 - [Installing](#installing)
 - [Wrapper Classes](#wrapper-classes)
    - [S3](#s3) - upload files to directly to an Amazon Simple Storage Service (S3) bucket. Your server must sign requests using a private key.
+   - [Azure](#azure) - upload files directly to Azure storage
```
Comment (rnicholus) [inline README.md:41]: Can you move this to the 1st item (to maintain ABC ordering)?

**Comment by tgandrews** [comment]
Is this sufficient?

**Review by rnicholus — CHANGES_REQUESTED** [review]
Otherwise looks good to me. Are you already using these changes in your app?

**Comment by tgandrews** [comment]
I've updated the order.

RE use: It's not fully integrated and in production yet. But, it works well in our PoC.

**Review by rnicholus — APPROVED** [review]

**Comment by rnicholus** [comment]
I think this can go out at 0.3.0 since 0.x.x already let developers know that this library is not yet stable. Thanks for your work on this. Can you also add a link to the [Azure feature page](http://docs.fineuploader.com/branch/master/features/azure.html) in the docs, similar to what I have done with the S3 section?

**Comment by tgandrews** [comment]
Updated the docs to include the azure feature page info and also included the server examples as well.

**Comment by rnicholus** [comment]
Awesome! Are you interested in becoming a collaborator on this project?

File: src/wrappers/azure.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import BaseWrapper from './base-wrapper'
+import qq from 'fine-uploader/lib/core/azure'
+import { traditional as callbackNames } from './callback-names'
```
Comment (rnicholus) [inline src/wrappers/azure.js:3]: Oh! One thing missing here we need a set of "azure" callback names. I'll fix and merge.

Comment (rnicholus) [reply src/wrappers/azure.js:3]: On second thought - that can wait until we actually _have_ some Azure-specific callbacks.

**Comment by tgandrews** [comment]
Awesome :tada:

Thanks for merging. I'm happy to help where I can especially regarding azure specific things and I'll let you know if we have any issues in prod

**Pull request merged**